nio/client/async_client.py

```diff
@@ -3431,3 +3435,52 @@ async def room_upgrade(self,
                 return RoomUpgradeError("Could update the new rooms aliases")
 
         return RoomUpgradeResponse(new_room.room_id)
+
+    @logged_in
+    async def has_event_permission(self,
+                                   room_id: str,
+                                   event_name: str,
+                                   event_type: str = "event") -> Union[bool, ErrorResponse]:
```

Comment:
and this one here is not the event_type, it's either  "event" or "state". Should this maybe be covered by a boolean "state", which is false by default?